crates/misc/py/examples/gcd/README.md

```diff
@@ -3,7 +3,7 @@
 To build `gcd.wasm` use rustc (nightly) for wasm32 target with debug information:
 
 ```
-rustc +nightly --target=wasm32-unknown-unknown -g gcd.rs --crate-type=cdylib
+rustup run nightly rustc --target=wasm32-unknown-unknown -g gcd.rs --crate-type=cdylib
```

Comment:
How come this was adjusted? I think the `rustc +channel` is a bit more terse and idiomatic than `rustup run`, although these should both be equivalent.

Reply:
The `rustc +channel ...` didn't work on my box (it just added another compilation unit `+channel`) I'd be fine with reverting this if it's a just my box thing

Reply:
Hm are you sure you have rust installed correctly? It sounds like you have a `rustc` in `PATH` which isn't rustup-managed?

Reply:
Ah, that's the reason. I installed the toolchain thru the mozilla nix overlay, so it's probably just the bare compiler. I'll revert this commit. https://github.com/mozilla/nixpkgs-mozilla